stdlib/event/time.lua

```diff
@@ -0,0 +1,73 @@
+--- Time Event module
+-- @module Event.Time
+
+require 'stdlib/event/event'
+
+Event.Time = {}
+Event.Time._last_change = {}
+
+--All times are offset by 0.5
+--This is because both EvoGUI and MoWeather already apply that offset.
+--Following the precedent to remain consistent.
+--Actually, this little snippet is even borrowed from EvoGUI.
+if remote.interfaces.MoWeather then
+    -- assume MoWeather's getdaytime is sane
+    function Event.Time.get_day_time(surface_name_or_index) return remote.call("MoWeather", "getdaytime", surface_name_or_index) end
```

Comment:
Feels weird to be explicitly calling another mods function, but on the other hand mod compatibility out of the box is a plus.